wordpress-objects.taxonomy.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+class Taxonomy {
+
+	static $taxonomy;
+
+	protected $_tax;
+
+	public function __construct( $taxonomy = null ) {
```

Comment:
@mattheu my general preferred method would be to use a static `get()` for an existing taxonomy. This means we don't need to use exception, but instead the static `get()` returns a WP_Error on get existing:

```PHP
class Taxonomy {
    public static function get( $tax ) {
        if ( ! get_taxonomy( $taxonomy ) ) {
            return new WP_Error( 'not-found', 'The taxonomy does not exist' );
        }
        return new Taxonomy( $tax );
    }
}
```